src/clusterfuzz/_internal/bot/tasks/update_task.py

```diff
@@ -193,9 +193,10 @@ def update_source_code():
   process_handler.cleanup_stale_processes()
   shell.clear_temp_directory()
 
-  root_directory = environment.get_value('ROOT_DIR')
-  temp_directory = environment.get_value('BOT_TMPDIR')
-  temp_archive = os.path.join(temp_directory, 'clusterfuzz-source.zip')
+  # ROOT_DIR just means the clusterfuzz directory.
+  source_root_directory = environment.get_value('ROOT_DIR')
+  root_directory = os.path.dirname(source_root_directory)
```

Comment:
Re-using the root_Directory variable here might be confusing. Can we call this something else?